src/IssueViz.Security.UnitTests/Hotspots/LocalHotspotStoreTests.cs

```diff
@@ -344,14 +344,14 @@ public void GetAll_ReviewedServerHotspots_Filters()
         var serverHotspot2 = CreateEmptyServerHotspot(status: "REVIEWED", resolution: "ACKNOWLEDGED");
         var serverHotspot3 = CreateEmptyServerHotspot(status: "REVIEWED", resolution: "FIXED"); //Expected to be filtered out
         var serverHotspot4 = CreateEmptyServerHotspot(status: "REVIEWED", resolution: "SAFE"); //Expected to be filtered out
-        serverStoreMock.Setup(x => x.GetAll()).Returns(new[] { serverHotspot1, serverHotspot2, serverHotspot3 });
+        serverStoreMock.Setup(x => x.GetAll()).Returns(new[] { serverHotspot1, serverHotspot2, serverHotspot3, serverHotspot4 });
 
-        var issueVis1 = Mock.Of<IAnalysisIssueVisualization>();
-        var issueVis2 = Mock.Of<IAnalysisIssueVisualization>();
-        var issueVis3 = Mock.Of<IAnalysisIssueVisualization>();
-        var issueVis4 = Mock.Of<IAnalysisIssueVisualization>();
-        var issueVis5 = Mock.Of<IAnalysisIssueVisualization>();
-        var issueVisualizations = new[] { issueVis1, issueVis2, issueVis3, issueVis4 };
+        var issueVis1 = CreateUniqueIssueViz();
+        var issueVis2 = CreateUniqueIssueViz();
+        var issueVis3 = CreateUniqueIssueViz();
+        var issueVis4 = CreateUniqueIssueViz();
+        var issueVis5 = CreateUniqueIssueViz();
+        var issueVisualizations = new[] { issueVis1, issueVis2, issueVis3, issueVis4, issueVis5 };
```

Comment:
The test wasn't passing `serverHotspot4` or `issueViz5` to the mocks, but it was checking for `issueViz5` in the `VerifyContent` call in line 365 and the test was passing...